### PR TITLE
Crowdin translation updates (integration branch)

### PR DESCRIPTION
**Pull request by johnjohndoe**
- This is the integration branch automatically created and updated by [Crowdin](https://crowdin.com).
- The branch is rebased irregularly by @johnjohndoe.

### CODE REVIEW

**Comment by johnjohndoe** [comment]
@matmair Why is there 110 commits? I received 100+ email from Travis CI. If every change triggers a build then Travis CI build points will be used up in a few days. Can we please change this?

**Comment by johnjohndoe** [comment]
@matmair Why does the branch contain files like `dimensions.xml`, `styles_congress.xml`, `attrs.xml` which do not contain translatable resources?

**Comment by johnjohndoe** [comment]
@matmair Are you still on this?

File: app/src/main/res/values-it/preferences.xml [hunk 1]
```diff
@@ -1,18 +1,76 @@
...
+    <string name="preference_summary_alternative_schedule_url">Inserire un URL di backup se il server preconfigurato non può essere raggiunto.</string>
+    <!-- App notification settings -->
+    <string name="preference_title_app_notification_settings">Personalizza le notifiche</string>
+    <string name="preference_summary_app_notification_settings">Open the associated system settings screen from here.</string>
     <!-- Insistent session alarms -->
-
+    <string name="preference_title_insistent_alarms_enabled">Abilita sveglia insistente</string>
+    <string name="preference_summary_insistent_alarms_enabled">Keeps an alarm on infinitely.</string>
+    <!-- Category Engelsystem -->
+    <string name="preference_summary_engelsystem_json_export_url"><![CDATA[Per vedere i tuoi turni inserisci qui il tuo <u>URL personale </u>.<br/><br/>
```
Comment (johnjohndoe) [inline app/src/main/res/values-it/preferences.xml:70]: Something seems to be wrong here:

```
/home/runner/work/EventFahrplan/EventFahrplan/app/src/main/res/values-it/preferences.xml:69:4: Failed to flatten XML for resource 'preference_summary_engelsystem_json_export_url' with error: Invalid unicode escape sequence in string
> Task :app:createCcc36c3DebugCompatibleScreenManifests
"{str}"
/home/runner/work/EventFahrplan/EventFahrplan/app/src/main/res/values-it/preferences.xml:69:4: string/preference_summary_engelsystem_json_export_url does not contain a valid string resource.
```

Comment (matmair) [reply app/src/main/res/values-it/preferences.xml]: @johnjohndoe should work now. Seems like it was a escaping problem with the italian l'

**Comment by johnjohndoe** [comment]
@matmair, as promised I restructured the commits on this branch to let it become easier to comprehend. I have a few questions left which you might be able to answer:

1. Can Crowdin be configured to ignore `placeholder.xml` in all languages?
2. Why is Crowdin also reformatting the source files (English)? See ebc150c39da046a8890b6626149627a8f8f9c249.
3. Can the commit message(s) in Crowdin be customized, e.g. when a new language is added in the backend, when a string is translated?

**Comment by johnjohndoe** [comment]
Superseded by #551.

**Pull request closed without merging**